tests/operators/test_read.py

```diff
@@ -178,6 +178,14 @@ def test_check_input_files_no_file_in_directory(tmp_path):
         read._check_input_files(tmp_path, "*")
 
 
+def test_um_normalise_callback_rename_stash(cube):
+    """Correctly translate from STASH to LFRic variable name."""
+    read._um_normalise_callback(cube, "m01s03i236", None)
```

Comment:
```suggestion
    read._um_normalise_callback(cube, None, None)
```
The field argument is unused in the function, so we shouldn't give it anything.